app/build.gradle

```diff
@@ -107,16 +107,16 @@ dependencies {
     implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
     implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
     implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
-    implementation 'com.squareup.okhttp3:okhttp:4.2.0'
-    implementation 'com.squareup.okhttp3:logging-interceptor:4.2.0'
+    implementation 'com.squareup.okhttp3:okhttp:4.2.2'
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.2.2'
     implementation 'org.conscrypt:conscrypt-android:2.2.1'
     implementation 'com.github.connyduck:sparkbutton:2.0.1'
     implementation 'com.github.chrisbanes:PhotoView:2.3.0'
     implementation 'com.mikepenz:google-material-typeface:3.0.1.3.original@aar'
     implementation('com.theartofdev.edmodo:android-image-cropper:2.8.0') {
         exclude group: 'com.android.support'
     }
-    implementation 'com.evernote:android-job:1.2.6'
+    implementation 'com.evernote:android-job:1.4.2'
```

Comment:
This now uses Workmanager under the hood. I wonder A) if this changes the reliability of notifications B) if we could also implement Workmanager directly